import-export-cli/integration/envSpecific_test.go

```diff
@@ -176,12 +175,42 @@ func TestEnvironmentSpecificParamsEndpointSecurityDigest(t *testing.T) {
 
 	apiParams := testutils.ReadAPIParams(t, args.ParamsFile)
 
-	validateEndpointSecurityDefinition(t, api, apiParams, importedAPI)
+	testutils.ValidateEndpointSecurityDefinition(t, api, apiParams, importedAPI)
 }
 
-func TestEnvironmentSpecificParamsEndpointSecurityBasic(t *testing.T) {
-	superTenantAdminUsername := superAdminUser
-	superTenantAdminPassword := superAdminPassword
+func TestEnvironmentSpecificParamsEndpointSecurityDigestDevopsTenant(t *testing.T) {
```

Comment:
Same here and please check the other places as well.

Reply:
Fixed via e4f37dba86efef403f4a1d3967821d1f2791019b